packages/tx/package.json

```diff
@@ -51,12 +51,16 @@
     "tsc": "../../config/cli/ts-compile.sh"
   },
   "dependencies": {
+    "@chainsafe/ssz": "^0.9.2",
     "@ethereumjs/common": "^3.0.2",
     "@ethereumjs/rlp": "^4.0.0",
     "@ethereumjs/util": "^8.0.3",
     "ethereum-cryptography": "^1.1.2",
     "ethers": "^5.7.1"
   },
+  "peerDependencies": {
+    "c-kzg": "^1.0.8"
+  },
```

Comment:
Ok, just to double check on how this works here from a user perspective a short (assumed) summary:

1. Peer dependencies are (still) not auto-installed by any Node version (correct?), so users have to install manually
2. Installation would be done with `npm install c-kzg` (or is there an npm included "Install all peer dependencies" command?)
3. As currently implemented users of the other tx types do not have to install any kzg library at all

All of the above correct?

Reply:
According to the [npm documentation](https://docs.npmjs.com/cli/v9/configuring-npm/package-json#peerdependencies), looks like peer dependencies do get installed by default with npm v7 so we may want to update this to use the `peerDependencyMeta` field instead since that makes them optional.  The idea here would be that you shouldn't need to install c-kzg unless you're specifically working with blob transactions.  Otherwise, there's no point in installing.

Reply:
I'm going to update package.json to specify it's an optional dependency via `peerDependenciesMeta` so it doesn't get automatically installed.  Then, we can update the readme to note you need to independently install c-kzg in order for this to work.

Reply:
Ah, yes, now I remember (the peer dependency npm 7 thing, since we had already the discussion around our own EthereumJS dependency structure).

That sounds like a good solution with this `peerDependenciesMeta` field 👍, just google on this in relation to npm 7 a bit and found the following extract from an npm RFC discussion [on the topic](https://github.com/npm/rfcs/discussions/289) ("Auto-installing peer dependencies in npm@7.
"):

> peerOptional deps are deps that are listed in peerDependencies and also have an entry in  peerDependenciesMeta containing {"optional": true}. These sorts of dependencies are not installed by default, but if installed, must be resolved to a version that satisfies the optional peer dependency relationship.

So seems this creates the behavior we would expect/want also on npm 7. 🙂